graphql/schema/types/filters.graphql

```diff
@@ -90,12 +90,10 @@ input PerformerFilterType {
 }
 
 input SceneMarkerFilterType {
-  """Filter to only include scene markers with this tag"""
-  tag_id: ID
   """Filter to only include scene markers with these tags"""
```

Comment:
Removed this as I couldn't find any usages of it, neither by searching for `TagId` nor `tag_id` (with "match word" semantics, otherwise it obviously matches the plural form). And using `tags` instead should yield the same result anyway.

If this is still needed I will readd it.

Reply:
For compatibility purposes, this should be left in. We can mark it as deprecated if it truly isn't needed, but for now keep it in.